nuget.config

```diff
@@ -5,6 +5,5 @@
     <add key="automatic" value="True" />
   </packageRestore>
   <packageSources>
-    <add key="NuGet" value="https://www.nuget.org/api/v2/" />
```

Comment:
NOTE: Removed this to avoid warning NU1507: There are 2 package sources defined in your configuration. When using central package management, please
map your package sources with package source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source. The following sources are defined: NuGet, nuget.org